src/components/controls.vue

```diff
@@ -60,9 +59,17 @@ export default {
     };
   },
   computed: {
-    controlItems() {
-      return flatten(Object.values(this.controls))
-        .filter(control => control.label.toLowerCase().includes(this.filterQuery.toLowerCase()))
+    controls() {
+      return this.nodeTypes
+        .filter(nodeType => nodeType.control)
+        .filter(nodeType => nodeType.label.toLowerCase().includes(this.filterQuery.toLowerCase()))
```

Comment:
I'm not sure why this is needed, especially as the filterQuery is an empty string. I tried removing this check and everything was broken though...

Reply:
`filterQuery` isn't always an empty string; it's updated using the `b-form-input` in the template.